### PR TITLE
Nexus: Print warning for the Nexus user before pwscf rsync

### PR DESCRIPTION
**Pull request by aannabe**
## Proposed changes

When a new NSCF is located on a different path than SCF, Nexus will rsync the SCF results into the new path. This is a great feature for submitting multiple NSCF calculations with various k-points. However, this can take quite a while and can give the false sense that Nexus is hanging when, in fact, it is silently rsync'ing the results. This has confused me a few times.

I simply put two print statements warning the user that this is happening. It only executes the first time when the directory does not exist. There is probably a better way to output this - let me know if there is and I will be happy to update.

### Does this introduce a breaking change?
- No

## What systems has this change been tested on?
All Nexus tests pass on my laptop (Linux Mint 21.3), except for 61/69 qdens_radial. 

## Checklist
- Yes. This PR is up to date with current the current state of 'develop'

### CODE REVIEW

**Review by prckent — DISMISSED** [review]
Good idea

File: nexus/lib/pwscf.py [hunk 1]
```diff
@@ -227,6 +227,7 @@ def incorporate_result(self,result_name,result,sim):
                 outdir = os.path.join(self.locdir,c.outdir)
                 command = 'rsync -av {0}/* {1}/'.format(result.outdir,outdir)
                 if not os.path.exists(outdir):
+                    print('Running rsync for the {} directory. This might take a while.'.format(outdir))
                     os.makedirs(outdir)
```
Comment (ye-luo) [inline nexus/lib/pwscf.py:231]: Please add a paired printout indicating the completion of rysnc.

Comment (aannabe) [reply nexus/lib/pwscf.py]: Done. Also moved the earlier print to just before the rsync. Makes more sense this way.

Comment (ye-luo) [reply nexus/lib/pwscf.py]: Please add back the printout before copy. They service distinct purpose and there is no harm to be verbose on the status of nexus.

**Comment by ye-luo** [comment]
Test this please

**Review by ye-luo — APPROVED** [review]

**Pull request merged**